src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -0,0 +1,245 @@
+import errno
+import time
+from mgr_module import MgrModule
+
+import orchestrator
+
+
+class NoOrchestrator(Exception):
+    pass
+
+
+class OrchestratorCli(MgrModule):
```

Comment:
I thought we couldn't do _ in the module names.. I guess we can?  We just have to translate to camelcaps?

Reply:
This is just the python convention of foo_bar module names and FooBar for class names.  The class name isn't really used anywhere.